uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1001.php

```diff
@@ -12,7 +12,6 @@ public function upgrade_1001() {
     $this->up1001_listDelete('ContactType', 'name', $this->up1001_civicrmContactTypesList());
     $this->up1001_listDelete('CaseType', 'name', $this->up1001_civicrmCaseTypesList());
     $this->up1001_listDelete('RelationshipType', 'name_b_a', $this->up1001_civicrmRelationshipTypesList());
-    $this->up1001_listDelete('LocationType', 'name', $this->up1001_civicrmLocationTypesList());
```

Comment:
can you remove the method itself 

https://github.com/civicrm/civihr/blob/474aa4a1603b616096716135ec3832306a3030ef/uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1001.php#L162

since it is not used anymore